test/verifiedTests/testTopology/testFindExtremePathway.m

```diff
@@ -46,14 +42,14 @@
     solverOK = changeCobraSolver(solverPkgs{k}, 'LP', 0);
 
     if solverOK == 1
-        v = findExtremePathway(model);
-        assert(all(abs(model.S * v) < tol))
-
-        % testing findExtremePathway with different arguments
-        if isfield(model, 'revRxns')
-            model = rmfield(model, 'revRxns');
-        end
-
+%         v = findExtremePathway(model);
```

Comment:
Ahh ok... My fault.
Yes, it doesn't work any more (since the removal of revRxns). at least if no lb/ub are defined. 
I'll move it further down in the function. after lb/ub are added, at which point it will work again.